### ISSUE TITLE
Audio is not Paused when App is Closed

### ISSUE THREAD

**Issue by canewsin** [Raj2503/Pacman-Flutter#5]
Tested on Android.

**Comment by Raj2503** [comment]
Yeah, I am aware of that. If you completely close the app from background applications, then only the audio stops. 
Otherwise if you just dismiss it, the audio continues

**Comment by Raj2503** [comment]
A suggestion would be to make the audio player a Flutter Background Service.
It will reduce the load on the foreground processing, helping with the frame rates.

Plus it will be easier to control and pause the audio in case of the app is dismissed by using AppLifeCycle.

**Comment by canewsin** [comment]
There is AppLifeCycle Widget, didn't remember the name, this may help to manage app state and audio control.

**Comment by Raj2503** [comment]
I guess this is what you are referring to [AppLifecycleState](https://api.flutter.dev/flutter/dart-ui/AppLifecycleState.html)
